docs/code/MlNetCookBook.md

```diff
@@ -737,13 +739,11 @@ var catColumns = data.GetColumn<string[]>(data.Schema["CategoricalFeatures"]).Ta
 
 // Build several alternative featurization pipelines.
 var pipeline =
-    // Convert each categorical feature into one-hot encoding independently.
-    mlContext.Transforms.Categorical.OneHotEncoding("CategoricalOneHot", "CategoricalFeatures")
```

Comment:
>OneHotEncoding [](start = 37, length = 14)

Why not leave this here? I think it was intended to show the difference between using this estimator with the `OutputKind` argument and without.